src/python-fastui/fastui/events.py

```diff
@@ -25,4 +25,25 @@ class BackEvent(BaseModel):
     type: Literal['back'] = 'back'
 
 
-AnyEvent = Annotated[Union[PageEvent, GoToEvent, BackEvent], Field(discriminator='type')]
+class ToastEvent(BaseModel):
```

Comment:
I think it would be better to implement toast as a standard component with an event to show it, like a modal.

Then you could either display a toast using a page event, or include a toast on a new page when you returned it with effectivley `show=True` so it appears immediately.